app/helpers/holdings_helper.rb

```diff
@@ -122,11 +129,15 @@ def status(item)
         'Unknown/Never'
       end
     else
-      @statuses[item[:status].to_s.downcase]
+      Status.find_by!(short_code: [item[:status].to_s.downcase]).name
     end
+  rescue ActiveRecord::RecordNotFound => e
+    Rollbar.error("Error retriving name for Status #{item[:status].to_s.downcase}", e)
+    Status.create(short_code: item[:status].to_s.downcase, name: 'Unknown')
```

Comment:
Same concern here if whether `item` is from us or from users. Makes me caution to see us creating something in on an exception if we can't find something in our database

Reply:
For sure.  In theory there should be a human readable string for any short code that Symphony could return, but that's not always the case.  We want to surface the error once so that we can track down why that might be the case and in the meantime have 'Unknown' shown in the interface to the user (rather than blowing up or showing some cryptic code).